### DISCUSSION TITLE
Welcome to Algo-Phantoms-Frontend Discussions!

### DISCUSSION THREAD

**Discussion by vatsalkesarwani12** [Algo-Phantoms/Algo-Phantoms-Frontend#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aniketsingh98571** [comment]
Is this website solely going to be built with only HTML, CSS/Bootstrap, and Javascript?

**Comment by Deepak-dex** [comment]
# Adding code editor

_hi I wanted to discuss about embedding ace code editor into algo-phantom_

 I'm thinking of embedding ace code editor for all languages and a codepen like editor for html ,css  and js.
